compiler/rustc_query_impl/src/plumbing.rs

```diff
@@ -300,6 +303,66 @@ pub(crate) fn create_query_frame<
     QueryStackFrame::new(name, description, span, def_kind, hash)
 }
 
+pub(crate) fn try_load_from_on_disk_cache<'tcx, Q>(tcx: TyCtxt<'tcx>, dep_node: DepNode)
```

Comment:
```suggestion
fn try_load_from_on_disk_cache<'tcx, Q>(tcx: TyCtxt<'tcx>, dep_node: DepNode)
```

Reply:
Can't be removed, unfortunately. `define_queries` is expanded at the crate root; it might be possible to change but I would rather not mix that with other changes. Let me know if I should make a PR changing it to expand in `plumbing.rs` and wait until it merges before following up here.

Reply:
oh no I'm wrong, this is only used from `query_callback`, not the macro. sorry for the noise.